<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddQuestionDialog</class>
 <widget class="QDialog" name="AddQuestionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Question adding</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Enter your question:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="questionTextInput">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Answer 1: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="answer1TextInput">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="answer1CorrectCheckBox">
             <property name="text">
              <string>Correct</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Answer 2: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="answer2TextInput">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="answer2CorrectCheckBox">
             <property name="text">
              <string>Correct</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Answer 3: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="answer3TextInput">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="answer3CorrectCheckBox">
             <property name="text">
              <string>Correct</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Answer 4: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="answer4TextInput">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="answer4CorrectCheckBox">
             <property name="text">
              <string>Correct</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Number of points: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="numberOfPointsSpinBox">
             <property name="maximumSize">
              <size>
               <width>92</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
